src/Controller/DocumentController.php

```diff
@@ -245,4 +257,25 @@ public function view(CaseEntity $case, Document $document, DocumentUploader $upl
 
         return $response;
     }
+
+    private function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setRequired('view_timezone')
+        ;
+    }
```

Comment:
Rather than adding a `view_timezone` option to `DocumentController` and passing it on to `DocumentType`, you could just add and set it in `DocumentType` using constructor argument injection from `services.yaml`.

Reply:
Good idea, i actually just passed it onto `CaseEventDocumentType` through `DocumentType`.

Used constructor argument injection to inject `view_timezone` to `CaseEventDocumentType`.